post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -247,6 +247,8 @@ protected void processCachedIds() {
                     case CASE_MANAGEMENT:
                         processTopic(keyTopic, entity, ids,
                                 investigationRepository::executeStoredProcForCaseManagement);
+                        processTopic(keyTopic, entity, ids,
+                                investigationRepository::executeStoredProcForFStdPageCase);
```

Comment:
For better logging it's better to use another signature of this method:
```
processTopic(keyTopic, entity.getName(), ids,
                                investigationRepository::executeStoredProcForFStdPageCase, "sp_f_std_page_case_postprocessing");
```
Otherwise, in the log it will indicate, that the `sp_nrt_case_management_postprocessing` was executed.